Update Ops Objectives for Q3 2025 (#11989)
PostHog/posthog.com: contents/teams/people/objectives.mdx

Talent

> * Make offers within 2 business days of Superday 

Reviewer: ```suggestion
* Make offers within 2 business days of SuperDay 
```

* Have a new team structure for a team of 6 in Talent
* Hire lots of new people - each list is in priority order:
  *  TBD

> Finance

Reviewer: ```suggestion
Finance - Janani
```

> * Get Transfer pricing in place for inter-company agreement

Reviewer: ```suggestion
* Get transfer pricing in place for inter-company agreement between US and UK
```

* Engage with new accountants in the US to ramp up before EOY 
* Engage with auditors for 2025 audit

> * Have new accountants in Germany working for us by end of quarter

Reviewer: ```suggestion
* Have new accountants in Germany working for us
```

* Have new revenue recognition process working 
* Make sure all our expenses are GAAP compliant

Legal & Compliance

> * Create a Trust Centre on Drata

Reviewer: ```suggestion
* Create a Trust Center on Drata
```

> * New UK Equity Plan Implementation

Reviewer: ```suggestion
* Implement new UK equity plan
```

> * Create Anti-Bribery and Anti-Corruption Policy within 90 days of closing funding

Reviewer: ```suggestion
* Create anti-bribery and anti-corruption policies as Series D post-closing item
```

> * Ensure all customer facing terms are ready for scaling 

Reviewer: What does this mean?

Author: Privacy policy, BAA, NDA, Terms, DPA would constitute our customer facing terms and all need various tweaks / changes to allow us to scale from an ease of use, flexibility from a commercial perspective and company protection perspective

Reviewer: I guess I'm struggling to understand what 'ready for scaling' actually means - is the aim here to basically put more stuff on autopilot so we're not manually approving/reviewing things like DPAs, NDAs etc.?

> * Have Intercompany Agreement in place 90 days after closing funding 

Reviewer: ```suggestion
* Have Intercompany Agreement in place as Series D post-closing item
```

People Ops 

* Reduce human input in the onboarding process to only PeopleOps, Team Lead, Onboarder

> * New process for booking off-sites & onboardings 

Reviewer: What does this mean?

Author: That we have a new process for booking off-site and onboardings by the end of the quarter

Reviewer: Yes, those are the words that are written, thank you 😂 

Why are we doing this? Is it to make it less distracting for team leads? Is the process to centralize it with ops? Or is this coming from somewhere else? Right now, it's not obvious why 'new process' is a useful thing to do here.

Reviewer: Cool yeah, I'd suggest a little rewrite to make it clear this is why we're doing it

* GTM offsite sorted
* Location booked for company offsite 2026

> * Be able to create new merch easier

Reviewer: ```suggestion
* Be able to create new merch with minimal outside involvement besides Lottie artwork
```

> * Make Merch work 

Reviewer: ```suggestion
* Complete Micromerch migration, with merch fulfilment running smoothly
```
